assets/js/admin/ppec-upgrade-notice.js

```diff
@@ -0,0 +1,39 @@
+;(function ( $, window, document ) {
+	'use strict';
+
+	// Check whether PayPal Payments is installed.
+	let is_paypal_payments_installed = false;
+	if ( $( 'tr[data-slug="woocommerce-paypal-payments"]' ).length ) {
+		is_paypal_payments_installed = true;
+	}
+
+	// Handle delete event for PayPal Payments.
+	$( document ).on( 'wp-plugin-delete-success', function( response ) {
+		setTimeout( function(){
+			if ( is_paypal_payments_installed && $( 'tr#woocommerce-paypal-payments-deleted' ).length ) {
```

Comment:
I might be wrong, but apparently `response` includes information about the deleted plugin(s). Would it be possible to use that instead of this check here? If it's too much trouble, we could leave it as is and try to find the "deleted" row to detect the deletion.

Reply:
The `response` is supposed to return the ID of the plugin deleted. However, the data parameter returns `undefined`. Hence I used this logic.

Here is a screenshot of the response object.

![Response](https://d.pr/i/DoVkAG+)

Reply:
Yes, the event doesn't include the data, but the second argument should. The callback could be something like this:

```js
	$( document ).on( 'wp-plugin-delete-success', function( ev, response ) {
		// `response` has the details.
	} );
```

If you can, please give it a try. If that still doesn't work for you or using the `response` objects proves to be too complicated, we can keep the current approach. Let me know :)

Reply:
Sure. I'll give it a try @jorgeatorres. 

Based on this, I only see one param - https://github.com/WordPress/wordpress-develop/blob/2382765afa36e10bf3c74420024ad4e85763a47c/src/js/_enqueues/wp/updates.js#L1057

Reply:
Yes, that's how `.trigger()` works. Because the parameter is not an event object, it is passed as an extra parameter. The callback, though, always receives the event and _then_ the rest of the extra parameters.

Reply:
Gotcha! Will try that :D

Reply:
Changes updated